fedimint-bin-tests/src/main.rs

```diff
@@ -544,13 +545,15 @@ async fn dev_fed(task_group: &TaskGroup, process_mgr: &ProcessManager) -> Result
     fed.await_gateways_registered().await?;
     info!("gateways registered");
     fed.use_gateway(&gw_cln).await?;
+    let electrs = Electrs::new(bitcoind.clone()).await?;
```

Comment:
let do this concurrently with Lnd::new and Lightningd::new.

Reply:
Sure!